Source/WebCore/platform/graphics/win/SimpleFontDataWin.cpp

```diff
@@ -130,9 +127,9 @@ RefPtr<Font> Font::platformCreateScaledFont(const FontDescription& fontDescripti
 
     LOGFONT winfont;
     GetObject(m_platformData.hfont(), sizeof(LOGFONT), &winfont);
-    winfont.lfHeight = -lroundf(scaledSize * (m_platformData.useGDI() ? 1 : 32));
+    winfont.lfHeight = -lroundf(scaledSize * 32);
```

Comment:
Could this 32 be given the same name as the 32 used earlier?  Fewer magic numbers.  Or at least a comment explaining what 32 means here.

Reply:
Good point. I didn't understand it. `* 32` is for CG fonts. WinCairo might should've used the useGDI code path. I should study before removing.

Reply:
Due to [cairo's bug](https://gitlab.freedesktop.org/cairo/cairo/-/issues/3), I can't remove the `* 32` at the moment. So, I added a constant `cWindowsFontScaleFactor`.